src/app/tests/suites/certification/Test_TC_CC_3_1.yaml

```diff
@@ -34,6 +34,15 @@ tests:
       response:
           value: 1
 
+    - label:
+          "Check current hue attribute value matched the value sent by the last
+          command"
+      disabled: true
+      command: "readAttribute"
+      attribute: "current hue"
```

Comment:
I would recommend to use CamelCase style only since it is the one use in the specs

Reply:
Tried as per spec but YAML supporting different style
![image](https://user-images.githubusercontent.com/84442963/148348711-7dc66dea-16e9-4aea-974b-3adc0a35e426.png)